spec/features/admin_can_create_a_place.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+feature "Admin can log in add a new place", %Q{
+  As an admin
+  I want to add a new place
+  So that I can share were to go place to go
+} do
+
+  let!(:user3) {User.create!(
+    id: 3,
+    first_name: "Chris",
+    last_name: "Fine",
+    email: "Cf@hotmail.com",
+    password: "password",
+    role: "admin"
+  )}
+
+  scenario 'admin is able to add a new place' do
+    visit new_user_session_path
```

Comment:
Please replace this logic (for signing in) with a sign in method, as described in https://launchboston25.slack.com/archives/CGV84UWFP/p1563904635004700

Reply:
do this for every one of these tests where your test currently has a user sign in

Reply:
Fixed this. Thank you! :)